src/statement_generator.cpp

```diff
@@ -962,7 +973,11 @@ unique_ptr<ParsedExpression> StatementGenerator::GenerateWindowFunction(optional
 		result->partitions.push_back(GenerateExpression());
 	}
 	if (RandomPercentage(30)) {
-		result->orders = std::move(GenerateOrderBy()->orders);
+		if (verification_enabled) {
+			result->orders = std::move(GenerateOrderByAll()->orders);
```

Comment:
we are only checking a window function here, so you don't need to add an order by